app/scripts/controllers/MainCtrl.js

```diff
@@ -1,11 +1,22 @@
 (function(){
-  function MainCtrl(Room, $scope){
+  function MainCtrl(Room, Message, $rootScope, $scope){
```

Comment:
At this point, I'd say MainCtrl might be doing too much. Time to split into another Ctrl for the room related logic?